README.md

```diff
@@ -1,8 +1,8 @@
-# nil: NIx Language server
+# nil: Nix Language server
```

Comment:
this seems intentional, the capitalized characters spell out nil

Reply:
:thinking:  Yeah. But it seems to be confusing for many ones. There's also https://github.com/oxalica/nil/pull/46#discussion_r1051548116
Not sure if there's a better way to express it.

Reply:
:sweat_smile: I didn't notice that the `s` is lowercase, so the only capitalized letters are `n` `i` `l` 

Egg on my face!

Reply:
I removed this diff, but will leave the PR open as it resolves unrelated typos